presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/env/DockerContainer.java

```diff
@@ -152,23 +160,30 @@ public DockerContainer withHealthCheck(Path healthCheckScript)
                 .withCreateContainerCmdModifier(command -> command.withHealthcheck(cmd));
     }
 
-    public Duration getStartupTime()
+    public synchronized Duration getStartupTime()
```

Comment:
why synchronized? AtomicLong?

Reply:
would it be better if i made this atomic?